packages/server/src/server.ts

```diff
@@ -21,6 +21,7 @@
 import { Relay, RelayImpl, JsonRpcError, predefined, MirrorNodeClientError } from '@hashgraph/json-rpc-relay';
```

Comment:
Nice, but my approach would be to make this validations in `koaJsonRpc` with the other checks. There we already have some kind of invalid params check, we can just extend it and not cluster `server.ts` with unnecessary code.
Overall looks good, it's important to have validations.

Reply:
Yeah, I wasn't sure for the best place for this.